src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,45 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE_ALLOWED = 18;
+    private static final int MAX_AGE_ALLOWED = 125;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
+    public static int getMinAgeAllowed() {
+        return MIN_AGE_ALLOWED;
+    }
+
+    public static int getMaxAgeAllowed() {
+        return MAX_AGE_ALLOWED;
+    }
```

Comment:
I don't think we need these methods here. If you need these constants outside RegistrationServiceImpl class, make them public